### DISCUSSION TITLE
Weighted Kernel Density

### DISCUSSION THREAD

**Discussion by AbelVM** [Turfjs/turf#2784] [Announcements]
Hi!

I've spent the morning porting and shaping the good ol' ESRI's **(weighted)Kernel Density** functionality into TURFjs.

I have made a new branch, but no PR because I'm not able to properly run the required linter or tests, so it surely needs some extra work. Something must be wrong with my setup, some combination of node, npm, babel, etc versions that had forced me to code almost blindly in terms of testing.

My local linter and tests look OK, but the workflow in the contribution guide is failing for me for the simplest test :disappointed: , so I haven't digged deeper with more complex tests.

The code lives here:

https://github.com/Turfjs/turf/tree/abelvm/kernelDensity/packages/turf-kernel_density

Any help is welcome! Virtual :beers: are cooling in the virtual fridge for the one who unblocks it!

It would help me to fix my local setup too, so I can make some other PRs that I have waiting in the queue

**Comment by rowanwins** [comment]
Hey @AbelVM 

Unfortunately due to #1428 now's not a great time to be contributing new modules. Hopefully once I get #1432 landed thing will become a fair bit simpler again.

So stay tuned :)
